scalac-scoverage-plugin/src/main/scala/scoverage/report/CoberturaXmlWriter.scala

```diff
@@ -39,7 +39,7 @@ class CoberturaXmlWriter(sourceDirectories: Seq[File], outputDir: File) extends
   }
 
   def klass(klass: MeasuredClass): Node = {
-    <class name={klass.name}
+    <class name={klass.fullClassName}
```

Comment:
I'm not sure what the Cobertura XML format requires here.
Is this change going to be a problem?

Do you have access to a Cobertura formatted XML report to check?